src/cli/validators.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+import jsonschema
+from jsonschema import Draft7Validator, RefResolver, draft7_format_checker
+from cli.exceptions import JSONSchemaValidationError
+from cli.utils import read_json_file
+
+
+def validate_json_string(instance_path: Path) -> None:
+    read_json_file(instance_path)
+
+
+def validate_json_schema(instance_path: Path, schema_path: Path) -> None:
```

Comment:
Why not copy it from qnerouter? there an issue was solved for checking schema's on different platforms.